### ISSUE TITLE
Nest: Enh: Lower "We received this event ... from Nest .. -> ignore" from DEBUG to TRACE

### ISSUE THREAD

**Issue by mrguessed** [openhab/openhab#2295]
I've spent some quality time in the logs diagnosing stuff and, at `DEBUG` Level, the Nest Binding emits messages like this:

```log
19:45:42.221 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestSmokeDownstairs_software_version', state='2.0rc5') from Nest, so we don't send it back again -> ignore!
19:45:42.224 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestSmokeUpstairs_software_version', state='2.0rc5') from Nest, so we don't send it back again -> ignore!
19:45:42.228 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestTStatDownstairs_has_fan', state='ON') from Nest, so we don't send it back again -> ignore!
19:45:42.232 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='Nest_time_zone', state='America/Los_Angeles') from Nest, so we don't send it back again -> ignore!
19:45:42.234 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestTStatDownstairs_humidity', state='55') from Nest, so we don't send it back again -> ignore!
19:45:42.238 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestTStatDownstairs_is_using_emergency_heat', state='OFF') from Nest, so we don't send it back again -> ignore!
19:45:42.242 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestSmokeGuestBedroom_smoke_alarm_state', state='ok') from Nest, so we don't send it back again -> ignore!
19:45:42.247 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestTStatDownstairs_software_version', state='4.3.3') from Nest, so we don't send it back again -> ignore!
19:45:42.250 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestSmokeDownstairs_is_online', state='ON') from Nest, so we don't send it back again -> ignore!
19:45:42.254 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestSmokeTopOfStairs_software_version', state='2.0rc5') from Nest, so we don't send it back again -> ignore!
19:45:42.258 DEBUG o.o.b.n.internal.NestBinding[:288]- We received this event (item='NestTStatDownstairs_ambient_temperature_c', state='19.0') from Nest, so we don't send it back again -> ignore!
```

I have 6x Nest Protects, and 2x Thermostats, so this generates a bunch of log traffic every minute.

Since these are all "ignore" level, and aren't making changes to the state of openHAB, it would be handy if they were lowered to `TRACE` level to aid the visibility of the real Nest changes.

PS: Awesome binding...

**Comment by watou** [comment]
I completely agree.  If Thomas does too, I will submit a pull request.  BTW, thanks! :)

**Comment by mrguessed** [comment]
He'll be up for it.  Again, you've got their attention, and this will be a
BIG feature on the 1.7 list when that release goes out.

If you have the cycles, you should just submit the PR.


BTW: I augmented your Nest Examples page, will Rules and some limited
colourization.  Hope you dont mind.

https://github.com/openhab/openhab/wiki/Nest-Binding-Example

**Comment by watou** [comment]
Thank you for adding the rule examples!  Very helpful.  I've added the decrease from debug to trace for the echo suppression log message to my nest-fixes pull request.

**Comment by watou** [comment]
This issue would be resolved with pull request https://github.com/openhab/openhab/pull/2294.

**Comment by mrguessed** [comment]
One quirk of this is that I'm now able to tell that the Nest Binding isn't logging actionable, and potentially data-modifying, _Commands_ in `DEBUG` mode.

So when I use a Rule with:
```xtend
Nest_away.sendCommand("away")
```

for an Item like:
```xtend
String   Nest_away "Away [%s]"                 {nest="=[structures(Home).away]"}
```

there's no output for any _Commands_ sent to the Nest Binding.  I didn't realize before due to the other log noise, but now that's been cleared out it's obvious.  Sorry, I should have checked earlier, but now I can see these messages are only delivered in `TRACE` mode:

```log
09:13:43.624 TRACE o.o.b.n.internal.NestBinding[:256]- internalReceiveCommand(item='Nest_away', command='away')
09:13:43.626 TRACE o.o.b.n.internal.NestBinding[:322]- About to set property 'structures(Home).away' to 'away'
```

It would be handy if this were promoted to `DEBUG`.  I can file a separate issue if you'd like, since it's technically different than my original request.

Bottom line, I'm looking to make any common user-debugging scenario work in `DEBUG` mode (since `TRACE` mode often emits sensitive data, and switching modes can't be done on the fly in Production deployments)

**Comment by watou** [comment]
Updated my nest-fixes branch / pull request #2294 to increase logging to debug for message Mark mentioned.

**Comment by mrguessed** [comment]
This is closed as a result of #2294 being merged.
